Modules/Filtering/FFT/test/itkComplexToComplexFFTImageFilterTest.cxx

```diff
@@ -137,16 +137,4 @@ itkComplexToComplexFFTImageFilterTest(int argc, char * argv[])
     std::cerr << "Unknown pixel type string." << std::endl;
     return EXIT_FAILURE;
   }
-
-  // Test streaming enumeration for ComplexToComplexFFTImageFilterEnums::TransformDirection elements
```

Comment:
Should this block be moved before if starting on line 107? Instead of being removed.

Reply:
@dzenanz Good question! I did not write the code, so honestly I don't know. But if I understand correctly, these ten lines of code just print two enum values. They don't really test any assertion. And then it does `return EXIT_FAILURE`. So whatever it printed, the test would always fail at this point. Personally, I think these lines of code may be removed.

Reply:
They were probably there to increase code coverage. I wouldn't mind restoring them to operational status.

Reply:
> I wouldn't mind restoring them to operational status.

As far as I can see, they never had operational status before  😸  Introduced with pull request #1471 (merged February last year), they were always unreachable: https://github.com/InsightSoftwareConsortium/ITK/blob/0dd31a62f851a0ae95ef4c6d6c85e531a4ebca59/Modules/Filtering/FFT/test/itkComplexToComplexFFTImageFilterTest.cxx#L147-L155

Anyway, if you want to, I can move these ten lines of code up, in order to make them reachable for the first time 😃

Reply:
😄 Yes, I would prefer that. And I assume @jhlegarreta would love it!

Reply:
If the aim is to test `operator<<`, I would prefer a test that inserts these two enum values into an `std::ostringstream`, instead of polluting the standard output! (And then doing test assertions on the result of `std::ostringstream`). But that's _way_ beyond the scope of this pull request. So I'll just move the printing code a few lines up for now.

Reply:
Your assumption is correct @dzenanz :+1: . @N-Dekker thanks for these cleanup. Yes, please, move the block so that it can be reached and the coverage guaranteed.

Reply:
> Yes, please, move the block so that it can be reached and the coverage guaranteed.

OK, I will, but I would like to emphasize that proper unit testing is more that just code coverage. It should check, preferably in an automatic way, that the result of function calls is according to their specification, or expectation.

Reply:
Of course. But even just invoking something can point out problems by failing to compile, or crashing at run time. Testing doesn't have to  be an all-or-nothing approach. If we try to make it that, I expect that we will be getting nothing a lot more often than all 😞

Reply:
> OK, I will, but I would like to emphasize that proper unit testing is more that just code coverage. It should check, preferably in an automatic way, that the result of function calls is according to their specification, or expectation.

Agreed. Another example of the need you point is in #983. I have failed to find the time and peace of mind to try to propose a solution during this time.

An issue can be opened concerning the enums' streaming checking. Bug fixes and PRs lead to finding new room for improvement, but IMHO trying to address many issues at a time makes things less easier or PRs to stall (and I do have some of these :worried:). So we all try our best in here in the time we have.